Updated interface of network connection module (#1214)
Azure/ResourceModules: arm/Microsoft.Network/connections/readme.md

| `lock` | string | `'NotSpecified'` | `[CanNotDelete, NotSpecified, ReadOnly]` | Specify the type of lock. |
| `peer` | object | `{object}` |  | The remote peer. Used for connection type [ExpressRoute] |
| `routingWeight` | int | `-1` |  | The weight added to routes learned from this BGP speaker. |
| `tags` | object | `{object}` |  | Tags of the resource. |
| `usePolicyBasedTrafficSelectors` | bool | `False` |  | Enable policy-based traffic selectors |
| `virtualNetworkGateway2` | object | `{object}` |  | The remote Virtual Network Gateway. Used for connection type [Vnet2Vnet] |
| `virtualNetworkGatewayConnectionType` | string | `'IPsec'` | `[IPsec, Vnet2Vnet, ExpressRoute, VPNClient]` | Gateway connection type. |
| `vpnSharedKey` | string | `''` |  | Specifies a VPN shared key. The same value has to be specified on both Virtual Network Gateways |

### Parameter Usage: `virtualNetworkGateway1`

The primary virtual network gateway object.

```json
"virtualNetworkGateway1" : {
    "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myRG/providers/Microsoft.Network/virtualNetworkGateways/myGateway",
}
```

### Parameter Usage: `virtualNetworkGateway2`

> The seconday virtual network gateway used for VNET to VNET connections.

Reviewer: ```suggestion
The secondary virtual network gateway used for VNET to VNET connections.
```

```json
"virtualNetworkGateway1" : {

>     "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myRG/providers/Microsoft.Network/virtualNetworkGateways/myGateway",

Reviewer: ```suggestion
    "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myRG/providers/Microsoft.Network/virtualNetworkGateways/myGateway02",
```

}
```

### Parameter Usage: `localNetworkGateway2`

The local virtual network gateway object.

```json

> "localNetworkGateway2" : {

Reviewer: ```suggestion
"localNetworkGateway1" : {
```

    "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myRG/providers/Microsoft.Network/localNetworkGateways/myGateway",
}
```

### Parameter Usage: `localNetworkGateway2`

> The local virtual network gateway object.

Reviewer: Please elaborate on the difference between this and the previous localNetworkGateway parameter usage

> ### Parameter Usage: `localNetworkGateway2`
> 
> The local virtual network gateway object.

Reviewer: Did you mean 
```suggestion
### Parameter Usage: `peer`

The remote peer object used for ExpressRoute connections
```
